source/fe/fe_poly_tensor.cc

```diff
@@ -166,8 +166,9 @@ namespace internal
 
 
 
-template <class PolynomialType, int dim, int spacedim>
-FE_PolyTensor<PolynomialType, dim, spacedim>::FE_PolyTensor(
+template <int dim, int spacedim>
+FE_PolyTensor<dim, spacedim>::FE_PolyTensor(
+  const TensorPolynomialsBase<dim> &polynomials,
   const unsigned int                degree,
```

Comment:
You can probably also get rid of the `degree` member variable (and constructor argument) since you can query that from the `poly_space` object.

Reply:
Good idea!